### PR TITLE
Upload and apply fixes

### LINKED ISSUE #1308: "Upload and Apply" Info Box Can Be Upsized and Resized
Upload and apply info box is the only one that can be upsized and rezised (look at the screenshot). 

Testflow: 
1. Login as kapua-sys
2. Connect live Kura device to Kapua
3. Go to Devices, select the live device and navigate to Configuration -> Snapshots
4. Click _**Upload and Apply**_
**It is possible to change windows size, which is not possible with any other window.**

uploadandapplywindowsedapovecat

Browsers used:
- Google Chrome (Version 63.0.3239.132 (Official Build) (64-bit)))
- Safari (Version 11.0.2 (13604.3.5))
- Mozilla Firefox (57.0.4 (64-bit))
- Opera (Version 50.0.2762.45)

### PR DESCRIPTION
**Pull request by bojanutkovic**
#1308, #1311 
Signed-off-by: Bojan Utkovic <bojan.utkovic@comtrade.com>

### CODE REVIEW

**Comment by LeoNerdoG** [comment]
I have verified that this PR solves issue #1308 and #1311 .

**Comment by muros-ct** [comment]
@bojanutkovic Can you update headers to 2017, 2018.

File: console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/util/Constants.java [hunk 1]
```diff
@@ -22,4 +22,6 @@ private Constants() {
     public static final int LABEL_WIDTH_USER_FORM = 445;
     public static final int LABEL_WIDTH_INFO = 150;
     public static final int LABEL_WIDTH_FORM_EXPLANATION = 600;
+    public static final int LABEL_WIDTH_UPLOAD_AND_APPLY = 75;
```
Comment (muros-ct) [inline console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/util/Constants.java]: Is this correct naming for constants as others are starting with type (LABLE for example) then metric (WIDTH) then context (USER) and then type again (FORM).
What is UPLADE_AND_APPLY?

**Comment by Coduz** [comment]
Please use the "Squash and merge" button when merging this to avoid commits like "PR fixes" to be on develop.

**Comment by muros-ct** [comment]
Tests are failing. Please run build without -DskipTests.

**Comment by muros-ct** [comment]
Tests are fixed on develop.

**Pull request merged**